google/auth/credentials.py

```diff
@@ -11,19 +11,20 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
-
 """Interfaces for credentials."""
 
 import abc
+import copy
 from enum import Enum
+import json
 import os
 
 from google.auth import _helpers, environment_vars
 from google.auth import exceptions
 from google.auth import metrics
 from google.auth._refresh_worker import RefreshThreadManager
 
+DEFAULT_TRUST_BOUNDARY = {"locations": [], "encoded_locations": "0x0"}
```

Comment:
I think this needs a warning that this dictionary should not be used directly, but copied from as a base template.